standards/X509-SVID.md

```diff
@@ -36,7 +36,9 @@ Perhaps the most important function of SPIFFE is to secure process to process co
 This specification addresses the encoding of SVID information into an X.509 certificate, the constraints which must be set, as well as how to validate X.509 SVIDs.
 
 ## 2. SPIFFE ID
-In an X.509 SVID, the corresponding SPIFFE ID is set as a URI type in the Subject Alternative Name extension (SAN extension, see [RFC 5280 section 4.2.16][2]). An X.509 SVID MUST contain exactly one URI SAN. It MAY contain any number of other SAN fields, including DNS SANs.
+In an X.509 SVID, the corresponding SPIFFE ID is set as a URI type in the Subject Alternative Name extension (SAN extension, see [RFC 5280 section 4.2.16][2]). An X.509 SVID MUST contain exactly one URI SAN, and by extension, exactly one SPIFFE ID. SVIDs containing more than one SPIFFE ID introduce challenges related to auditing and authorization logic. Validators encountering an SVID containing more than one SPIFFE ID MUST reject the SVID. Please see the [Validation](#5-validation) section for more information.
```

Comment:
This says the SVID must contain exactly one URI SAN, but then when talking about validation says the SVID must contain exactly one SPIFFE ID.

These two musts are not quite equivalent, as URI SANs might contain other (non-spiffe) kinds of URIs.

So, what's the motivation for restricting the SVID to a single URI SAN, as opposed to saying something like, a SVID must contain only one URI SAN with the scheme set to "spiffe" ?

If we really do want to restrict the SVID to a single URI SAN, then let's keep that language consistent for the validation.

Reply:
```suggestion
In an X.509 SVID, the corresponding SPIFFE ID is set as a URI type in the Subject Alternative Name extension (SAN extension, see [RFC 5280 section 4.2.16][2]). An X.509 SVID MUST contain exactly one URI SAN, and by extension, exactly one SPIFFE ID. SVIDs containing more than one SPIFFE ID introduce challenges related to auditing and authorization logic, and SVIDs containing more than one URI SAN introduce challenges related to SPIFFE ID validation. Validators encountering an SVID containing more than one URI SAN MUST reject the SVID. Please see the [Validation](#5-validation) section for more information.
```